packages/functional-tests/tests/misc.spec.ts

```diff
@@ -1,69 +1,48 @@
 import { test, expect } from '../lib/fixtures/standard';
 
-test.describe('severity-1', () => {
```

Comment:
I'm slightly confused here... We have a `test.describe("severity-1", ...)` wrapper here which will let us glob by "severity-1" keyword, presumably. THis PR seems to remove that and bumps the tests up to top level. 
Do we not need to filter by "severity-1" anymore, or am I misreading something?

Reply:
Yes you are right. I am going to add the wrapper again here.